README.md

```diff
@@ -446,6 +446,36 @@ var table = ReactDOM.render(
 These can be useful if you want to roll your own filtering input field
 outside of Reactable.
 
+You can also provide your own custom filtering functions:
+
+```jsx
+<Table className="table" id="table" data={[
+    {'State': 'New York', 'Description': 'this is some text', 'Tag': 'new'},
+    {'State': 'New Mexico', 'Description': 'lorem ipsum', 'Tag': 'old'},
+    {'State': 'Colorado',
+     'Description': 'new description that shouldn\'t match filter',
+     'Tag': 'old'},
+    {'State': 'Alaska', 'Description': 'bacon', 'Tag': 'renewed'},
+]}
+filterable={[
+    {
+        column: 'State',
+        filterFunction: function(contents, filter) {
+            // return true if the state contains 'x' and the filter
```

Comment:
I'd prefer to use a more real-world example for this. How about case-insensitive filtering?